src/app/app.component.html

```diff
@@ -1,16 +1,30 @@
 <div class="d-flex flex-column w-100">
 
-	<app-navbar [(toggleResults)]="showResults" [(toggleSummary)]="showSummary" (user)="doShowUser(true)" (report)="doShowReport(true)" [isLogged]="isLogged"></app-navbar>
+	<app-navbar
+		[(toggleResults)]="showResults"
+		[(toggleSummary)]="showSummary"
+		(user)="doShowUser(true)"
+		(report)="doShowReport(true)"
+		[isLogged]="isLogged"
+		[allFilesProcessed]="allFilesProcessed">
+	</app-navbar>
 
 	<div class="container">
 		<p></p>
-		<file-drop class="no-print" (onFileDrop)="fileDrop($event)">
+		<file-drop *ngIf="!readingFiles()" class="no-print" (onFileDrop)="fileDrop($event)">
 			<div class="text-center text-primary">
 				<i class="fa fa-upload" style="font-size:23px;"></i>
 				<p>Drop your allure files here</p>
 			</div>
 		</file-drop>
 
+		<div *ngIf="readingFiles()" class="progress allure-progress-bar">
```

Comment:
Can you please use Nanobar? Please see systelab components dialog-header

Reply:
I used the progress bar based in bootstrap in the same way that it's in systelab-components demo in utils page. 
To use Nanobar I have to add the dependency to systelab-charts and use it explicity in the code. If we want to use the nanobar as progress bar standard we should create a systelab-component progress bar as a wrapper in case we want to change it easily in the future.